daemon/commit.go

```diff
@@ -265,7 +276,17 @@ func (daemon *Daemon) Commit(name string, c *backend.ContainerCommitConfig) (str
 }
 
 func (daemon *Daemon) exportContainerRw(container *container.Container) (io.ReadCloser, error) {
-	if err := daemon.Mount(container); err != nil {
+	rwlayer, err := daemon.layerStore.GetRWLayer(container.ID)
+	if err != nil {
+		return nil, err
+	}
+	defer func() {
+		if err != nil {
+			daemon.layerStore.ReleaseRWLayer(rwlayer)
+		}
+	}()
+	err = daemon.Mount(container)
```

Comment:
Why not? I think these mount calls are needed for getting data of container's RWLayer if this container is exited.